src/Reveal.Sdk.Dom/Visualizations/ImageVisualization.cs

```diff
@@ -5,10 +5,24 @@
 
 namespace Reveal.Sdk.Dom.Visualizations
 {
+    /// <summary>
+    /// The image visualization is used for displaying images, PDFs, or web content.

```

Comment:
is PDFs correct? I haven't confirm this is supported

Reply:
Yes, this is the visualization being used to display PDFs